u1.hh

```diff
@@ -14,10 +14,12 @@ public:
   explicit _u1() : a(0) {}
   explicit _u1(double _a) : a(_a) {}
   _u1(const _u1& U) : a(U.a) {}
+  _u1(Complex _a) : a(std::arg(_a)) {}
```

Comment:
minor comment:
since the argument is not edited in the body of the function, is more efficient to call it as a constant reference:
``` c++
_u1(const Complex & _a) : // ... 
```